scripts/build-schema.php

```diff
@@ -6,9 +6,9 @@
 // update.php will call init.php
 require $install_dir . '/includes/sql-schema/update.php';
 
-$file = $install_dir . '/misc/db_schema.yaml';
```

Comment:
this doesnt look right?

Reply:
I must have accidentally let the IDE rename the variable.